app/services/no_bp_measure_service.rb

```diff
@@ -58,6 +61,7 @@ def execute_sql(period)
         AND "medical_histories"."deleted_at" IS NULL
         AND "medical_histories"."hypertension" = :hypertension
         AND "patients"."assigned_facility_id" in :facilities
+        AND "patients"."status" != :exclude_status
```

Comment:
@rsanheim It seems like github-ds doesn't support Postgres yet. I was hoping to do a `NOT IN ('dead', 'migrated')`. Might try writing this in Activerecord, if that works out we'll be able to use the scope as well.

Reply:
I tried interpolating the `('dead', 'migrated')` list like
```
AND "patients"."status" NOT IN :excluded_statuses
```
That didn't work out because it inserts single quotes around the brackets.